app/controllers/mixins/manager_controller_mixin.rb

```diff
@@ -100,7 +100,6 @@ def new
     def edit
       assert_privileges("#{privilege_prefix}_edit_provider")
       @explorer = true if explorer_controller?
-      @server_zones = Zone.visible.in_my_region.order('lower(description)').pluck(:description, :name)
       @server_zones = Zone.visible.in_my_region.order(Zone.arel_table[:name].lower).pluck(:description, :name)
```

Comment:
Why do we have arel here? Shouldn't this be just simply `order(:name)`? cc @kbrock

Reply:
Nevermind